configs/core/cytomineconfig.groovy.sample

```diff
@@ -32,9 +32,9 @@ grails.retrieval.enabled=$RETRIEVAL_ENABLED
 grails.backup.enabled=$BACKUP_ENABLED
 grails.bioformat.enabled=$BIOFORMAT_ENABLED
 grails.iipJP2.enabled=$IIP_JP2_ENABLED
-grails.software.enabled=$SOFTWARE_ENABLED
```

Comment:
$SOFTWARE_ENABLED and $SOFTWARE_DEVELOPMENT have a different purpose.

$SOFTWARE_ENABLED allows to disable Cytomine software for a (production) instance
$SOFTWARE_DEVELOPMENT means that software-related containers are run locally from the host machine instead of Docker

Obviously, SOFTWARE_DEVELOPMENT=true implies SOFTWARE_ENABLED=true